classes/class-wc-connect-order-item-meta.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Order_Item_Meta' ) ) {
+	class WC_Connect_Order_Item_Meta {
```

Comment:
I had the same thought - I inquired in Slack about it, and core WC will only delete options/data if that's opted in.

A very unscientific sampling of a few shipping extensions did not yield examples of total data annihilation on uninstall.

The issue this PR addresses only mentions `wp_options` in scope.